src/raftstore/store/util.rs

```diff
@@ -127,32 +127,37 @@ pub fn conf_change_type_str(conf_type: eraftpb::ConfChangeType) -> &'static str
 
 const MAX_WRITE_BATCH_SIZE: usize = 4 * 1024 * 1024;
 
+/// Return a vector which indicates the cf should be flushed or not for all cfs in the DB.
 pub fn delete_all_in_range(
     db: &DB,
     start_key: &[u8],
     end_key: &[u8],
     use_delete_range: bool,
-) -> Result<()> {
+) -> Result<Vec<bool>> {
     if start_key >= end_key {
-        return Ok(());
+        return Ok(vec![]);
     }
 
-    for cf in db.cf_names() {
-        delete_all_in_range_cf(db, cf, start_key, end_key, use_delete_range)?;
+    let cf_names = db.cf_names();
+    let mut need_flush_mem_table = Vec::with_capacity(cf_names.len());
```

Comment:
s/need_flush_mem_table/exist_data_before_cleanup
`delete_all_in_range` has nothing to do with flush memtable.